pymongo/helpers.py

```diff
@@ -80,6 +80,8 @@ def _index_list(key_or_list, direction=None):
     Takes such a list, or a single key, or a single key and direction.
     """
     if direction is not None:
+        if not isinstance(key_or_list, str):
+            raise TypeError("Expected a string and a a direction")
```

Comment:
"and a a direction" -> "and a direction"

Reply:
Fixed